test/integration/spec/connect.js

```diff
@@ -990,6 +1013,73 @@ describe('connect', function() {
           assert.equal(loPriTrackPub.track.isSwitchedOff, true, 'low pri tracks were not switched off');
           assert.equal(hiPriTrackPub.track.isSwitchedOff, false, 'hi pri track were switched off');
         });
+
+        it('should switch off RemoteVideoTracks that are not published by the Dominant Speaker', async () => {
+          const [aliceTracks, bobTracks] = await Promise.all([1, 2].map(async () => [
+            createSyntheticAudioStreamTrack() || await createLocalAudioTrack({ fake: true }),
```

Comment:
why not always `createLocalAudioTrack`?

Reply:
@makarandp0 Because we want to create a synthetic audio track, since in Travis and Jenkins, you don't necessarily have any sound in the default audio track.